<launch>

    <arg name="config_file" default="$(find cwru_davinci_moveit_object_handling)/config/collision_objects_generator.yaml"/>

    <rosparam ns="moveit_object_handling" file="$(arg config_file)"/>

    # name of the node
    <arg name="node_name" default="davinci_moveit_object_message_generator"/>

    ###############################################
    # Start the node which will publish moveit::CollisionObject messages.
    ###############################################
    <node name="$(arg node_name)" pkg="cwru_davinci_moveit_object_handling" type="davinci_moveit_object_message_generator" respawn="true" output="screen"/>
</launch>